src/StarcounterClientFiles/wwwroot/sys/palindrom-error-catcher/wct.conf.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
For the next releases, we could add `wct.conf.json` and `Gruntfile.js` to bower ignores